src/js/main.js

```diff
@@ -0,0 +1,3 @@
+import '../styles/style.css'
+const Arepa = 32;
+console.log(Arepa);
```

Comment:
Let's please try to not commit 'test code'. I know you added this while testing if the installation was successful, but this variable brings no value to the repository. It's also written in spanish and the idea is to have all the code written in english

If you want to add something to show everything works as expected, consider adding doing following instead:

```suggestion
const app = document.getElementById("app")
app.innerHTML = "e-commerce"
```